application/src/main/java/org/togetherjava/tjbot/commands/moderation/ModerationUtils.java

```diff
@@ -372,4 +372,37 @@ static Optional<TemporaryData> computeTemporaryData(String durationText) {
      */
     record TemporaryData(Instant expiresAt, String duration) {
     }
+
+    /**
+     * Gives out advice depending on the {@link ModerationAction} and the parameters passed into it.
+     *
+     * @param action gives out a {@link ModerationAction} to be able to respond and build a message
```

Comment:
i dont understand this sentence. maybe just `@param action the action that is being performed, such as banning an user`

Reply:
Done